extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -59,6 +59,6 @@ require([
 		title: videoDetails.title
 	});
 
-	featuredVideoTracking(playerInstance);
+	featuredVideoTracking(playerInstance, willAutoplay);
```

Comment:
what about third argument?

Reply:
Not needed here, we will use a default category if it's not passed.